client/landing/stepper/declarative-flow/internals/steps-repository/importer/index.tsx

```diff
@@ -97,18 +104,26 @@ export function withImporterWrapper( Importer: ImporterCompType ) {
 			dispatch( requestSites() );
 		}, [ dispatch ] );
 
-		useEffect( () => {
-			! siteId && site?.ID && setSiteId( site.ID );
-		}, [ siteId, site ] );
 		useAtomicTransferQueryParamUpdate( siteId );
 		useEffect( fetchImporters, [ siteId ] );
 		useEffect( checkFromSiteData, [ fromSiteData?.url ] );
 		useEffect( () => onComponentUnmount, [] );
 
-		if ( ! importer ) {
-			stepNavigator.goToImportCapturePage?.();
-			return null;
-		}
+		useEffect( () => {
+			if ( ! isLoading && ! site ) {
+				logToLogstash( {
+					feature: 'calypso_client',
+					tags: [ 'importer', importer, 'error' ],
+					error: 'Importer missing site info',
+					message: 'Importer missing site',
+					site_id: siteId,
+					site_slug: siteSlug,
+					properties: {
+						env: config( 'env_id' ),
```

Comment:
It looks like the `config` you imported is breaking the build.

Maybe we need two aspects to capture the environment?
 * Use the broader config package: `import config from '@automattic/calypso-config';`
 * Also capture `process.env.NODE_ENV`
```suggestion
						env_id: config( 'env_id' ),
						env: process?.env?.NODE_ENV,
```

Reply:
Oh thanks!
Fixed here [444259d](https://github.com/Automattic/wp-calypso/pull/102277/commits/444259d3cfd26dce119db009acfbc1f35016a1aa)